[2010-03-23 21:49:41,314] INFO     0[main](AssertMain.java:17) - Results asserter starting
[2010-03-23 21:49:41,317]DEBUG     3[main](AssertMain.java:20) - Program arguments:
[2010-03-23 21:49:41,322]DEBUG     8[main](AssertMain.java:22) - args[0] 'cipher-throw-wsinterceptorexception-sin'
[2010-03-23 21:49:41,322]DEBUG     8[main](AssertMain.java:22) - args[1] 'D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin'
[2010-03-23 21:49:41,323]DEBUG     9[main](AssertMain.java:45) - Building context
[2010-03-23 21:49:41,339]TRACE    25[main](AssertMain.java:66) - Context:
[2010-03-23 21:49:41,340]TRACE    26[main](AssertMain.java:68) - resultDir: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin
[2010-03-23 21:49:41,340]TRACE    26[main](AssertMain.java:68) - clientSoap: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin\soap-trace-client.log
[2010-03-23 21:49:41,341]TRACE    27[main](AssertMain.java:68) - serverLog: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin\log-server.log
[2010-03-23 21:49:41,341]TRACE    27[main](AssertMain.java:68) - config: cipher-throw-wsinterceptorexception-sin
[2010-03-23 21:49:41,341]TRACE    27[main](AssertMain.java:68) - serverSoap: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin\soap-trace-server.log
[2010-03-23 21:49:41,342]TRACE    28[main](AssertMain.java:68) - clientLog: D:\dev\stepframework\src_test\extensions-testbed\results\20100323\2135\cipher-throw-wsinterceptorexception-sin\log-client.log
[2010-03-23 21:49:41,342]DEBUG    28[main](AssertMain.java:76) - Entering assert dispatch
[2010-03-23 21:49:41,342]DEBUG    28[main](AssertMain.java:79) - Asserting configuration 'cipher-throw-wsinterceptorexception-sin'
[2010-03-23 21:49:41,362] INFO    48[main](Assertions.java:52) - assertClientSOAPFaultException
[2010-03-23 21:49:41,385] INFO    71[main](Assertions.java:162) - assertSOAPCounts
[2010-03-23 21:49:41,385] INFO    71[main](Assertions.java:118) - assertCountClientSOAPEnvelopes
[2010-03-23 21:49:41,387] INFO    73[main](Assertions.java:123) - assertCountServerSOAPEnvelopes
[2010-03-23 21:49:41,388] INFO    74[main](Assertions.java:128) - assertCountClientSOAPFaults
[2010-03-23 21:49:41,388] INFO    74[main](Assertions.java:133) - assertCountServerSOAPFaults
[2010-03-23 21:49:41,389] INFO    75[main](Assertions.java:172) - assertSOAPBodyCipherCounts
[2010-03-23 21:49:41,389] INFO    75[main](Assertions.java:138) - assertCountClientSOAPBodyCipher
[2010-03-23 21:49:41,390] INFO    76[main](Assertions.java:143) - assertCountServerSOAPBodyCipher
[2010-03-23 21:49:41,390] INFO    76[main](AssertMain.java:234) - SUCCESS! All assertions executed successfully for configuration 'cipher-throw-wsinterceptorexception-sin'
[2010-03-23 21:49:41,390] INFO    76[main](AssertMain.java:37) - End of results asserter.
